src/widget.tsx

```diff
@@ -397,7 +397,8 @@ export namespace DaskDashboardLauncher {
 
   export const DEFAULT_ITEMS = [
     { route: 'individual-graph', label: 'Graph' },
-    { route: 'individual-load', label: 'Load' },
+    { route: 'individual-nbytes', label: 'Memory Use' },
+    { route: 'individual-nprocessing', label: 'Processing Tasks' },
```

Comment:
@ian-r-rose the "Load" plot was split into two upstream.  I found that I needed to update these manually.  I know that at one point we discussed getting this information from Dask itself.  I'm inclined to leave this as future work, but wanted to point it out here.

Reply:
Yeah, I had wanted to do that, but was stymied by the CORS issues. Once we move some logic into a server-side proxy we should be able to serve the individual routes dynamically, but for now I think this is the only way.